samples/samples/snippets_test.py

```diff
@@ -781,3 +781,25 @@ def test_list_database_roles(capsys, instance_id, sample_database):
     snippets.list_database_roles(instance_id, sample_database.database_id)
     out, _ = capsys.readouterr()
     assert "new_parent" in out
+
+
+@pytest.mark.dependency(name="create_table_with_foreign_key_delete_cascade")
+def test_create_table_with_foreign_key_delete_cascade(capsys, instance_id, sample_database):
+    snippets.create_table_with_foreign_key_delete_cascade(instance_id, sample_database.database_id)
+    out, _ = capsys.readouterr()
+    assert "Created Customers and ShoppingCarts table with FKShoppingCartsCustomerId" in out
+
+
+@pytest.mark.dependency(name="alter_table_with_foreign_key_delete_cascade",
+                        depends=["create_table_with_foreign_key_delete_cascade"])
```

Comment:
Hi, As we are adding `Foreign Key Delete Cascade` constraint , shouldn't is should **Not** depend on sample which has already added the constraints while creating the tables ?

Reply:
It is common in samples to have this dependency to reduce the time to run these